libraries/Microsoft.Bot.Builder.LanguageGeneration/Events/ExpressionContext.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace Microsoft.Bot.Builder.LanguageGeneration
+{
+    /// <summary>
+    /// Expression container with source range.
+    /// </summary>
+    public class ExpressionContext
```

Comment:
ExpressionRef or ExpressionContext? ExpressionContext gives more feeling like this is a context used to evaluate expression?